server/routes/api/columns.js

```diff
@@ -46,7 +46,7 @@ router.post(
 // @access private
 router.put(
   "/update/:id",
-  [auth, columnValidationRules(), validate],
+  [auth, validate],
```

Comment:
Currently columnValidationRules() requires the "name" prop to be passed down but because I handle the saving of columns in the drag function with ids, I did not pass down the "name" prop.